samples/msal-browser-samples/HybridSample/views/client-redirect.hbs

```diff
@@ -6,7 +6,7 @@
     const msalInstance = new msal.PublicClientApplication({
         auth: {
             clientId: "{{clientId}}",
-            redirectUri: "https://localhost:3000/auth/client-redirect",
```

Comment:
Hey @markmnl do you mind also turning off the nativebroker in system config too on line 20. Just change the value of allowNativeBroker to false. Thank you!